### PR TITLE
feat: add typescript paths to the boilerplate

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,7 +87,9 @@
     "rimraf": "^5.0.1",
     "standard-version": "^9.5.0",
     "supertest": "^6.3.3",
-    "ts-jest": "^29.0.5",
+    "ts-jest": "^29.2.5",
+    "tsc-alias": "^1.8.10",
+    "tsconfig-paths": "^4.2.0",
```
Comment (netanelC) [inline package.json:92]: It seems that you are not using this package in the repo

**Review by netanelC — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "release": "standard-version",
     "test": "npm run test:unit && npm run test:integration",
     "prebuild": "npm run clean",
-    "build": "tsc --project tsconfig.build.json  && npm run assets:copy",
+    "build": "tsc --project tsconfig.build.json && npx tsc-alias -p tsconfig.build.json && npm run assets:copy",
```
Comment (netanelC) [inline package.json:19]:
```suggestion
    "build": "tsc --project tsconfig.build.json && tsc-alias -p tsconfig.build.json && npm run assets:copy",
```
Unnecessary

**Review by netanelC — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (CptSchnitz) [reply package.json]: fixed

File: package.json (hunk 1 above)
Comment (CptSchnitz) [reply package.json]: fixed

**Review by shimoncohen — APPROVED** [review]

**Review by netanelC — APPROVED** [review]

**Pull request merged**